src/migrations/tasks/20240205135550_twu-proposal-member.ts

```diff
@@ -0,0 +1,84 @@
+import { makeDomainLogger } from "back-end/lib/logger";
+import { console as consoleAdapter } from "back-end/lib/logger/adapters";
+import Knex from "knex";
+
+const logger = makeDomainLogger(consoleAdapter, "migrations");
+
+export async function up(connection: Knex): Promise<void> {
+  /**
+   * Adds a new column to the twuProposalMember table to store the TWUResource
+   * ids
+   */
+  await connection.schema.alterTable("twuProposalMember", function (table) {
+    table
+      .uuid("twuResource")
```

Comment:
Can we call it "resource" since twu is implied? Similar to the existing proposal column.